prometheus-to-sd/main.go

```diff
@@ -111,6 +114,23 @@ func main() {
 	<-make(chan int)
 }
 
+func getSourceConfigs(gceConfig *config.GceConfig) (sourceConfigs []config.SourceConfig) {
+	if *asDaemonSet {
```

Comment:
If asDaemonSet flag disables functionality of normal sources, you should add validation that forbids passing both